### PR TITLE
Avatar OnError

### PR DESCRIPTION
**Pull request by akoushke**

### CODE REVIEW

File: src/components/WebexAvatar/__snapshots__/WebexAvatar.test.js.snap [hunk 1]
```diff
@@ -241,3 +241,5 @@ exports[`Webex Avatar component matches snapshot with "typing" person status 1`]
   type="typing"
 />
 `;
+
+exports[`Webex Avatar component matches snapshot with wrong personID 1`] = `[Function]`;
```
Comment (adamweeks) [inline src/components/WebexAvatar/__snapshots__/WebexAvatar.test.js.snap]: Snapshot looks wrong here.

File: src/components/hooks/usePerson.js [hunk 2]
```diff
@@ -14,7 +14,12 @@ export default function usePerson(personID) {
 
   useEffect(() => {
     const onError = (error) => {
-      throw error;
+      setPerson({
+        displayName: ' ',
```
Comment (lalli-flores) [inline src/components/hooks/usePerson.js]: What happens if you set displayName to `null` or `undefined`?

Comment (akoushke) [reply src/components/hooks/usePerson.js]: nothing gets rendered

**Comment by akoushke** [comment]
> Could you squash the commits?

yes i did

File: src/components/hooks/usePerson.js (hunk 2 above)
Comment (adamweeks) [inline src/components/hooks/usePerson.js]: Could we pass alt text to say something like "Error: this is the error message"? I know it wasn't defined in the JIRA, but might be beneficial. Not a blocker though.

Comment (akoushke) [reply src/components/hooks/usePerson.js]: I think the log should be good enough.

Comment (lalli-flores) [reply src/components/hooks/usePerson.js]: You should have set this to empty string `''`, not space `' '`

Comment (lalli-flores) [reply src/components/hooks/usePerson.js]: I like the alt idea, something like `Unknown user`

**Pull request merged**